src/lib.rs

```diff
@@ -22,12 +31,49 @@
     trivial_numeric_casts,
     unreachable_pub,
     unused,
-    variant_size_differences
+    variant_size_differences,
+
+    clippy::pedantic,
+    clippy::nursery,
```

Comment:
As mentioned, pedantic and nursery are not desired lints. There's a reason clippy specifically has a separate grouping for them with allow by default enabled.